matchms/exporting/save_as_json.py

```diff
@@ -9,6 +9,23 @@ def save_as_json(spectrums: List[Spectrum], filename: str):
 
     :py:attr:`~matchms.Spectrum.losses` of spectrum will not be saved.
 
+    Example:
+
+    .. code-block:: python
+
+        from matchms import Spectrum
```

Comment:
I am getting numpy not found error when I run this.
```suggestion
        import numpy
        from matchms import Spectrum
```